### DISCUSSION TITLE
Notes August 31st

### DISCUSSION THREAD

**Discussion by lseelenbinder** [maplibre/maplibre#103] [Governing Board Meetings]
On August 31st, for the first time, the newly-elected governing board met. We discussed multiple topics, and took the following decisions.

- After an invitation from CARTO to attend their Deck.gl-focused event in Madrid in September, we discussed the best attendee, and decided that Birk should attend, if CARTO sponsors or he wishes to self-sponsor.
- We discussed sponsorship, especially about tracking both potential and current sponsors. It was determined that we need a pipeline and documentation to track the process. A ticket was created: #102.
- The remainder of the time was spent discussion budgets, hiring plans, and long-term goals of the budget and any paid team members. It was decided that the project needs a "lead" director or coordinator to manage the organization and to take charge of the many responsibilities of managing the resources we have received. This individual will be responsible for proposing budgets and hiring plans, which the board will approve.

All members of the board were present.
- @nyurik
- @klokan
- @birkskyum
- @HarelM
- @lseelenbinder

**Comment by botic** [comment]
Great to see the progress and the long-term possibilities.
